src/app/settings/components/settings/settings-menu/settings-menu.component.ts

```diff
@@ -1,11 +1,33 @@
-import { Component } from '@angular/core'
+import { Component, OnDestroy, OnInit } from '@angular/core'
+import { NavigationEnd, Router } from '@angular/router'
+import { Subject, takeUntil } from 'rxjs'
 import { SettingsPaths } from 'src/app/settings/settings-routing.module'
+import { ShelfPaths } from 'src/app/settings/shelf/shelf.module'
 
 @Component({
   selector: 'sofie-settings-menu',
   templateUrl: './settings-menu.component.html',
   styleUrls: ['./settings-menu.component.scss'],
 })
-export class SettingsMenuComponent {
+export class SettingsMenuComponent implements OnInit, OnDestroy {
   public settingsPaths = SettingsPaths
+  public shelfPaths = ShelfPaths
+  public isShelfRouteActive: boolean = false
+
+  private readonly unsubscribe$: Subject<void> = new Subject<void>()
```

Comment:
Minor thing: You don't need to declare the diamon brackets on the right side of equal sign because you have defined on the left side.

Reply:
Also do we want to use the`unsubscribe$` naming convention? If so we haven't been doing that in other files, but i see that we have it in a couple now.

Reply:
Ok, I will remove it.
@RasmusAlbrektsen unsubscribe$ name is really unique and we will change verry easy when we decide the naming

Reply:
@RasmusAlbrektsen are you reffering to the `unsubscribe` part or the `$` part?
Personnally, I think the name `unsubscribe` is a quite fitting name.

The `$` is a common convention in Angular to indicate the variable is a Subject/Observable. Personnally, I would prefer something like `unbsubscribeSubject` over `unsubscribe$`.